create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -424,12 +427,23 @@ private Contributor generateContributorFromAuthorTp(AuthorTp author, AuthorGroup
 
     private Optional<Map<String, String>> getOrganizationLabels(AuthorGroupTp authorGroup) {
         var organizationNameOptional = getOrganizationNameFromAuthorGroup(authorGroup);
-        return organizationNameOptional.map(organizationName -> Map.of(getLanguageIso6391Code(),
+        return organizationNameOptional.map(organizationName -> Map.of(getLanguageIso6391Code(organizationName),
                                                                        organizationName));
     }
 
-    private String getLanguageIso6391Code() {
-        return "en";
+    private String getLanguageIso6391Code(String textToBeGuessedLanguageCodeFrom) {
+        var detector = new OptimaizeLangDetector().loadModels();
```

Comment:
Optimaize is a bit corny 🌽

Reply:
Yeah, I tried to check if the newer version of tika just had changed the spelling to Optimize, but it seems the whole package is gone. (despite this is being an copy-paste example from their 2.3.0 documentation)